src/modeler/main.cpp

```diff
@@ -35,21 +33,20 @@ using namespace Antares::Optimisation::LinearProblemMpsolverImpl;
 using namespace Antares;
 using namespace Antares::Solver;
 using namespace Antares::Optimisation::LinearProblemApi;
-using namespace Antares::Optimisation::LinearProblemDataImpl;
 
 class SystemLinearProblem
```

Comment:
Another refactoring to do later would be to "empty" main() and move code in library. It would be more testable and usable in other components

Reply:
I agree, it would be nice to have one struct/class with all modeler inputs ([example of code duplication](https://github.com/AntaresSimulatorTeam/Antares_Simulator/pull/2671/files#diff-e459b7fc3f521838da6235795415d8a558c0c49f0f11c26e5d5f8976fcc4bc9a) that would be avoided)